### DISCUSSION TITLE
Train model For Persian

### DISCUSSION THREAD

**Discussion by masoudMZB** [snakers4/silero-models#48] [Ideas]
## ❓ I've read your repo. thanks for this nice project.

Where is the code which I can use to train model with my own data?
Can I know your Model Structure?

And How was the quality of data you used? I mean noises, transcription accuracy for training data, hours etc...

Thanks

**Comment by snakers4** [comment]
> Where is the code which I can use to train model with my own data?
> And How was the quality of data you used? I mean noises, transcription accuracy for training data, hours etc...

For a number of reasons at this particular moment decided not to provide this information.
If you would like to have a model for a low-resource language added or some domain added so that models are more robust there, we can do it pro-bono as a service for community.

> And How was the quality of data you used? I mean noises, transcription accuracy for training data, hours etc...

I was thinking about making a wiki page about this, but then I decided against it.
We decided to go for extended metrics section instead.

> Can I know your Model Structure?

The models, their code and the publications they are based on are public.
Please see our articles on the Gradient - https://thegradient.pub/.

**Comment by snakers4** [comment]
> Train model For Persian

We are not planning to have a EE (-edition) yet in languages we do not speak  properly (i.e. Hindi, Chinese, Arabic, Tagalog, etc), so if you would like to collaborate on making and sharing an СE-model with the community, you are welcome to ping me personally in telegram (@snakers41) or via email.
 
Actually we are doing the same for the Ukrainian language.

**Comment by masoudMZB** [comment]
> if you would like to collaborate on making and sharing an СE-model with the community, you are welcome to ping me personally in telegram (@snakers41) or via email.

Thanks for your kindness, These days I'm so busy I can't collaborate But I really appreciate your attitude.
But it is my honor to say a hello to you in telegram :)



> The models, their code and the publications they are based on are public.
Please see our articles on the Gradient - https://thegradient.pub/.

Thanks for the link.

**Comment by snakers4** [comment]
Thanks you for kind words
I will close this issue then
